src/tir/transforms/storage_access.cc

```diff
@@ -94,6 +94,21 @@ void StorageAccessVisitor::VisitStmt_(const EvaluateNode* op) {
   allow_append_ = false;
 }
 
+void StorageAccessVisitor::VisitStmt_(const LetStmtNode* op) {
+  allow_append_ = true;
+  curr_stmt_.access.clear();
```

Comment:
```suggestion
```
This should work. You can remove this line since the assertion `ICHECK_EQ(curr_stmt_.access.size(), 0U);` holds

Reply:
thank you for your review, the unecessary clear has been removed